R/summarize_ref_date_forecasts.R

```diff
@@ -0,0 +1,141 @@
+#' Summarize forecast hub data for a specific reference
+#' date. This function generates a tibble of forecast data
```

Comment:
The first line becomes the header in R function descriptions, so it is worth having a line break.
```suggestion
#' date. 
#' 
#' This function generates a tibble of forecast data
```